src/app/screens/Detail/components/CommentarySection/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import { CommentListProps } from '@interfaces/commentaries';
+
+import Comment from './components/Comment';
+import { SHOW_BUTTON } from './constants';
+import styles from './styles';
+
+const CommentarySection = ({ data }: CommentListProps) => {
+  const [showAll, setShowAll] = useState(false);
+  const list = showAll ? data : data.slice(0, 2);
+
+  return (
+    <View style={styles.container}>
+      {list.map(item => (
+        <Comment key={item.id} item={item} />
+      ))}
```

Comment:
I think you can fix the format here and you can merge these 3 lines into 1.

_A comment, in this case this function is taking just 3 lines and that's why is okay but if this functions gets bigger you should extract it._

Reply:
```suggestion
      {list.map(item => <Comment key={item.id} item={item} />)}
```

Reply:
The linter is auto-fixing this. I can't change it